exchanges/bitstamp/bitstamp_test.go

```diff
@@ -871,100 +865,120 @@ func TestWsRequestReconnect(t *testing.T) {
 		"data": ""
 	}`)
 	err := b.wsHandleData(pressXToJSON)
-	if err != nil {
-		t.Error(err)
-	}
+	assert.NoError(t, err, "WsRequestReconnect should not error")
 }
 
 func TestBitstamp_OHLC(t *testing.T) {
 	start := time.Unix(1546300800, 0)
 	end := time.Unix(1577836799, 0)
-	_, err := b.OHLC(context.Background(), "btcusd", start, end, "60", "10")
-	if err != nil {
-		t.Fatal(err)
+	o, err := b.OHLC(context.Background(), "btcusd", start, end, "60", "10")
+	assert.NoError(t, err, "TestBitstamp_OHLC should not error")
+	assert.Equal(t, "BTC/USD", o.Data.Pair, "Pair should be correct")
+	for _, req := range o.Data.OHLCV {
+		assert.Positive(t, req.Low, "Low should be positive")
+		assert.Positive(t, req.Close, "Close should be positive")
+		assert.Positive(t, req.Open, "Low should be positive")
+		assert.Positive(t, req.Volume, "Volume should be positive")
+		assert.NotEmpty(t, req.Timestamp, "Time should not be empty")
 	}
 }
 
 func TestBitstamp_GetHistoricCandles(t *testing.T) {
 	pair, err := currency.NewPairFromString("BTCUSD")
 	if err != nil {
-		t.Fatal(err)
+		t.Error("Invalid pair")
 	}
 	start := time.Unix(1546300800, 0)
 	end := time.Unix(1577836799, 0)
-	_, err = b.GetHistoricCandles(context.Background(), pair, asset.Spot, kline.OneDay, start, end)
-	if err != nil {
-		t.Fatal(err)
+	c, err := b.GetHistoricCandles(context.Background(), pair, asset.Spot, kline.OneDay, start, end)
+	assert.NoError(t, err, "GetHistoricCandles should not error")
+	assert.Equal(t, currency.NewPairWithDelimiter("BTC", "USD", ""), c.Pair, "Pair should be correct")
+	assert.NotEmpty(t, c, "Candles should not be empty")
+	for _, req := range c.Candles {
+		assert.Positive(t, req.High, "High should be positive")
+		assert.Positive(t, req.Low, "Low should be positive")
+		assert.Positive(t, req.Close, "Close should be positive")
+		assert.Positive(t, req.Open, "Low should be positive")
+		assert.Positive(t, req.Volume, "Volume should be positive")
+		assert.NotEmpty(t, req.Time, "Time should not be empty")
 	}
 }
 
 func TestBitstamp_GetHistoricCandlesExtended(t *testing.T) {
 	pair, err := currency.NewPairFromString("BTCUSD")
 	if err != nil {
-		t.Fatal(err)
+		t.Error("Invalid pair")
```

Comment:
Refer to other comment, looks like there's 4 of these. Or you could create a global test pair var with BTCUSD to be used for these and drop the need for NewPairFromString

Reply:
Let me know if this is something you were interested to do in this PR or not